docs/SKILL_SYNC_SETUP.md

```
@@ -0,0 +1,236 @@
+**OpenHands doesn't find the skill** — confirm the submodule is initialized and `SKILL.md` is at `.agents/skills/merge-prepping/SKILL.md` (not nested deeper):
+
+```bash
+ls .agents/skills/merge-prepping/SKILL.md
+git submodule status
+```
+
+If the path is nested (e.g. `.agents/skills/merge-prepping/plugins/.../SKILL.md`), Step 1 wasn't applied — the root restructure is required.
+
+**File exists but the skill is still "unknown"** — suspect the frontmatter, not the path, and check it in this order:
+
+1. **Is the YAML valid at all?** One-line repro: `python3 -c "import yaml; yaml.safe_load(open('SKILL.md').read().split('---')[1])"`. An **unquoted `description:` containing `: ` (colon-space)** — e.g. an inline example like `` `scope: smoother thing doing` `` — is a YAML mapping indicator and kills the whole parse (`mapping values are not allowed here`), silently dropping the skill. Quote the value (single quotes; double any internal apostrophes). Field-tested 2026-08-09: this was the real reason `prepping` failed to load even after its rename.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)